dashboard/src/index.css

```
@@ -149,6 +149,68 @@
+  color: hsl(var(--destructive));
+}
+
+[data-maibot-activity-orbit='true'][data-state='loading'],
+[data-maibot-activity-orbit='true'][data-state='unknown'] {
+  color: hsl(var(--muted-foreground));
+  opacity: 0.5;
+}
+
+[data-maibot-activity-orbit='true'] > span {
+  position: absolute;
+  border: 3px solid color-mix(in srgb, currentColor 32%, transparent);
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)